### ISSUE TITLE
Future not working properly

### ISSUE THREAD

**Issue by yashvadhvani** [laverdet/node-fibers#383]
```
const Future = require('fibers/future')
function myfunc() {
    var future = new Future();
    Eos().getInfo((err, res) => {
        future["return"]=res;
    })
    return future.wait();
};
console.log(myfunc());
```
The Error is can't wait without a fiber please help me with this

**Comment by laverdet** [comment]
This is correct. You need to wrap the whole thing in a fiber or `Future(...).detach()`
